packages/main/src/Tokenizer.ts

```diff
@@ -229,8 +248,14 @@ class Tokenizer extends UI5Element {
 		});
 	}
 
+	async popoverListFocusOut() {
```

Comment:
When you have one long token and its popover is open, if you delete the token from the popover, for a second an empty popover is shown and then the popover is closed at all. Can we prevent that?

Reply:
cant reproduce that